training/src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,126 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.aspectj.lang.annotation.RequiredTypes;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    public Book() {
+        this.id = null;
+    }
+
+
+    public Book(Long id, String author, String image, String title, String subtitle, String publisher,
+        String year, Integer pages, String isbn) {
+        this.id = id;
+        this.author = author;
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    @Column
```

Comment:
It's not necessary to define @Column if you're not going to change any of the default values. Instance variables get picked up by default ;)